src/Npgsql/Npgsql.csproj

```diff
@@ -25,7 +25,7 @@
   </ItemGroup>
 
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' Or '$(TargetFramework)' == 'net8.0' ">
```

Comment:
This references also System.Runtime.CompilerServices.Unsafe and System.Diagnostics.DiagnosticSource for net8.0, which we don't need. Can you please split the System.Text.Json reference to a different ItemGroup to not do that?